src/autolabel/models/anthropic.py

```diff
@@ -23,13 +23,15 @@ class AnthropicLLM(BaseModel):
         "claude-2.1": (8 / 1_000_000),
         "claude-3-opus-20240229": (15 / 1_000_000),
         "claude-3-sonnet-20240229": (3 / 1_000_000),
+        "claude-3-haiku-20240307": (0.25 / 1_000_000),
```

Comment:
do they also have a non timestamped name which always points to the latest model like there is gpt-4-turbo and gpt-4-1106-preview

Reply:
I do not think so. I tried testing that but it is not the case. Even langchain uses these.